### ISSUE TITLE
obs doesn't open

### ISSUE THREAD

**Issue by AntonioBerna** [obsproject/obs-studio#10763]
### Operating System Info

Other

### Other OS

Manjaro Linux

### OBS Studio Version

30.1.2

### OBS Studio Log URL

i don't access to this information because OBS doesn't start

### Expected Behavior

obs start normally

### Current Behavior

I run `obs` command on terminal and I obtain:

```
debug: Found portal inhibitor
debug: Attempted path: share/obs/obs-studio/locale/en-US.ini
debug: Attempted path: /usr/share/obs/obs-studio/locale/en-US.ini
debug: Attempted path: share/obs/obs-studio/locale.ini
debug: Attempted path: /usr/share/obs/obs-studio/locale.ini
debug: Attempted path: share/obs/obs-studio/themes
debug: Attempted path: /usr/share/obs/obs-studio/themes
debug: Attempted path: share/obs/obs-studio/themes/Yami.qss
debug: Attempted path: /usr/share/obs/obs-studio/themes/Yami.qss
warning: [Safe Mode] Unclean shutdown detected!
warning: [Safe Mode] User elected to launch normally.
info: Using EGL/X11
info: CPU Name: Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
info: CPU Speed: 3838.682MHz
info: Physical Cores: 4, Logical Cores: 8
info: Physical Memory: 15743MB Total, 8666MB Free
info: Kernel Version: Linux 6.6.32-1-MANJARO
info: Distribution: "Manjaro Linux" Unknown
info: Desktop Environment: XFCE (xfce)
info: Session Type: x11
info: Window System: X11.0, Vendor: The X.Org Foundation, Version: 1.21.1
info: Qt Version: 6.7.1 (runtime), 6.7.0 (compiled)
info: Portable mode: false
info: OBS 30.1.2-1 (linux)
info: ---------------------------------
info: ---------------------------------
info: audio settings reset:
	samples per sec: 48000
	speakers:        2
	max buffering:   960 milliseconds
	buffering type:  dynamically increasing
info: ---------------------------------
info: Initializing OpenGL...
info: Loading up OpenGL on adapter Intel Mesa Intel(R) UHD Graphics 620 (KBL GT2)
info: OpenGL loaded successfully, version 4.6 (Core Profile) Mesa 24.0.8-manjaro1.1, shading language 4.60
info: ---------------------------------
info: video settings reset:
	base resolution:   1920x1080
	output resolution: 1920x1080
	downscale filter:  Bicubic
	fps:               30/1
	format:            NV12
	YUV mode:          Rec. 709/Partial
info: NV12 texture support enabled
info: P010 texture support not available
info: Audio monitoring device:
	name: Default
	id: default
info: ---------------------------------
warning: Failed to load 'en-US' text for module: 'decklink-captions.so'
warning: Failed to load 'en-US' text for module: 'decklink-output-ui.so'
libDeckLinkAPI.so: cannot open shared object file: No such file or directory
warning: A DeckLink iterator could not be created.  The DeckLink drivers may not be installed
warning: Failed to initialize module 'decklink.so'
info: [pipewire] No captures available
warning: v4l2loopback not installed, virtual camera disabled
info: VAAPI: API version 1.21
info: FFmpeg VAAPI H264 encoding supported
Segmentation fault (core dumped)

```

### Steps to Reproduce

1. I use `obs` command on terminal
2. I obtain errors and obs doesn't open 
3.
...

### Anything else we should know?

if I decide not to open obs with the terminal but use the gui I get this popup. whatever I select OBS doesn't start

![Screenshot_2024-06-01_21-04-31](https://github.com/obsproject/obs-studio/assets/60107274/09195537-cff7-4053-a3eb-4c39bf0f63c9)

**Comment by Chiitoo** [comment]
Might want to try running with a new user to make sure it's not caused by some odd configuration files, and also via 'gdb', and then use the 'bt' command to get a backtrace to include here for some hints where it is happening.

**Comment by AntonioBerna** [comment]
> Might want to try running with a new user to make sure it's not caused by some odd configuration files, and also via 'gdb', and then use the 'bt' command to get a backtrace to include here for some hints where it is happening.

Hi, thanks for your reply. I was about to close the issue since I found the problem. Basically I noticed that I didn't have the `libva-nvidia-driver` package installed for the NVIDIA drivers and this was conflicting with OBS. Once the package was installed, everything went back to working as usual. Practically a few days ago I added the drivers of my video scheme to my Linux computer and I'm still trying to fix all the problems. Good continuation.
